tools/src/test/java/com/nedap/archie/rmobjectvalidatortest/RmObjectValidatorTest.java

```diff
@@ -40,6 +46,43 @@ public void requiredAttributesShouldBePresent() throws Exception {
         assertEquals("There should be 0 errors", 0, validationMessages.size());
     }
 
+    @Test
+    public void testEmptyElementWithoutArchetype() {
+        Element element = new Element();
+        RMObjectValidator rmObjectValidator = new RMObjectValidator(ArchieRMInfoLookup.getInstance());
+        List<RMObjectValidationMessage> messages = rmObjectValidator.validate(element);
+        assertEquals(2, messages.size());
+        for(RMObjectValidationMessage message:messages) {
+            assertTrue(Sets.newHashSet("/name", "/archetype_node_id").contains(message.getPath()));
+            assertEquals(RMObjectValidationMessageType.REQUIRED, message.getType());
+        }
+    }
+
+    @Test
+    public void testClusterWithoutArchetype() {
+        Cluster cluster = new Cluster();
+        cluster.setName(new DvText("test cluster"));
+        cluster.setArchetypeNodeId("id12");
+        Element element = new Element();
+        cluster.setItems(Lists.newArrayList(element));
+        RMObjectValidator rmObjectValidator = new RMObjectValidator(ArchieRMInfoLookup.getInstance());
+        List<RMObjectValidationMessage> messages = rmObjectValidator.validate(cluster);
+        assertEquals(2, messages.size());
+        for(RMObjectValidationMessage message:messages) {
+            assertTrue(message.getPath(), Sets.newHashSet("/items[1]/name", "/items[1]/archetype_node_id").contains(message.getPath()));
+            assertEquals(RMObjectValidationMessageType.REQUIRED, message.getType());
+        }
+    }
+
+
+    @Test
+    public void testNestedEmptyElementWithoutArchetype() {
+        Element element = new Element();
+        RMObjectValidator rmObjectValidator = new RMObjectValidator(ArchieRMInfoLookup.getInstance());
+        List<RMObjectValidationMessage> validate = rmObjectValidator.validate(element);
+        assertFalse(validate.isEmpty());
+    }
+
```

Comment:
Maybe also add a positive test case, i.e. a validation of an RM object without archetype that is valid.